spec/ffi/callback_spec.rb

```diff
@@ -500,27 +507,39 @@ class S8F32S32 < FFI::Struct
     expect(v).to eq(-1)
   end
 
+  def testCallbackU8rV(value, &block)
+    v1 = 0xdeadbeef
+    LibTest.testCallbackU8rV(value) { |i| v1 = yield(i) }
+    expect(v1).to eq(value)
+
+    # Using a FFI::Function (v2) should be consistent with the direct callback (v1)"
+    v2 = 0xdeadbeef
+    fun = FFI::Function.new(:void, [:uchar]) { |i| v2 = yield(i) }
+    LibTest.testCallbackU8rV(fun, value)
+    expect(v2).to eq(value)
+  end
+
   it ":uchar (0) argument" do
     v = 0xdeadbeef
```

Comment:
You can remove this and the `expect` line below as well, can't you?

Reply:
Good catch, indeed.

Reply:
This is fixed now (I originally wrote the helper a bit differently, but the latest version seems the simplest).